### ISSUE TITLE
Create All Submissions GQL permission doesn't work

### ISSUE THREAD

**Issue by M-Ro** [verbb/formie#412]
**Description**
Hi there,

We're encountering an issue when creating and submitting a form. For context, this is a headless scenario, forms are queried and then submitted via GraphQL.

![image](https://user-images.githubusercontent.com/433127/112981441-1aa0f000-9153-11eb-83cb-19144851f353.png)

With the 'Create All Submissions' permission checked, there doesn't actually seem to be ample permissions to create a submission, with the GQL api returning the response 'Unable to perform this action'.
Enabling the individual 'Create Submissions for form X' seems to work fine, though this causes issues with our automation workflow. Presumably the 'Create All Submissions' would act as a catch all for all forms.

![image](https://user-images.githubusercontent.com/433127/112981646-65226c80-9153-11eb-8629-3fa769df8eb3.png)

Schema is public, if it matters.

**Steps to reproduce**
1. Create a form
2. Fire a submission from the frontend

### Form settings
- Multi-page form: No
- Submission Method: GraphQL
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.3.25
- Craft version: 3.5.19.1
- Multi-site: Yes

Cheers

**Comment by daltonrooney** [comment]
Confirmed, I am having the same issue.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.25",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.26](https://github.com/verbb/formie/releases/tag/1.3.26)
